src/micro_http/src/common/headers.rs

```diff
@@ -343,14 +367,12 @@ mod tests {
     #[test]
     fn test_try_from_headers() {
         // Valid headers.
-        assert_eq!(
-            Headers::try_from(
-                b"Last-Modified: Tue, 15 Nov 1994 12:45:26 GMT\r\nContent-Length: 55\r\n\r\n"
-            )
-            .unwrap()
-            .content_length,
-            55
-        );
+        let headers =  Headers::try_from(
```

Comment:
Can we also have a test for text/plain and for no accept header to check the default?

Reply:
I thought that this unit test should check success and error cases for `Headers::try_from`. Parsing the accept headers is unit tested for `parse_header_line`, used by this method. Added test cases for text/plain and for no accept header to be extra sure.